de.metas.salescandidate.base/src/main/java-gen/de/metas/ordercandidate/model/X_C_OLCand.java

```diff
@@ -274,6 +274,89 @@ public int getBill_User_ID ()
 		return ii.intValue();
 	}
 
+	@Override
+	public org.compiere.model.I_C_BP_BankAccount getC_BP_BankAccount()
```

Comment:
do we still need this column? afaiu it became obsolete since we have a dedicated REST EP for payments now

Reply:
Right. Dropped it